quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -170,6 +171,11 @@ public void parseAndRun(List<String> cliArgs) {
                 addCommandOptions(cliArgs, cl);
             }
 
+            result = null;
+            if (Optional.ofNullable(currentCommand).map(AbstractCommand::getName).filter(Start.NAME::equals).isPresent()) {
```

Comment:
Couldn't we just test if `cuurentCommand instanceof Start`?

Reply:
That would be fine as long as we don't ever refactor to have something like StartDev be a subclass of Start.